c/zss.c

```diff
@@ -93,6 +93,26 @@ static JsonObject *readPluginDefinition(ShortLivedHeap *slh, char *pluginIdentif
 static WebPluginListElt* readWebPluginDefinitions(HttpServer* server, ShortLivedHeap *slh, char *dirname);
 static JsonObject *readServerSettings(ShortLivedHeap *slh, const char *filename);
 static InternalAPIMap *makeInternalAPIMap();
+ 
+ //redo
+  static int keyCompare(void *key1, void *key2) {
```

Comment:
The comment should be informative or not there at all. I think you just forgot to remove them. Also, you should ask Sean if there is a maximum plugin identifier length. strcmp() is ok, but it would be great to use strncmp() incase for some strange reason your input is not null terminated.